src/graph_gui.cpp

```diff
@@ -223,8 +223,8 @@ struct BaseGraphWindow : Window {
 		current_interval.lowest =  (11 * current_interval.lowest) / 10;
 
 		/* Always include zero in the shown range. */
-		double abs_lower  = (current_interval.lowest > 0) ? 0 : (double)abs(current_interval.lowest);
-		double abs_higher = (current_interval.highest < 0) ? 0 : (double)current_interval.highest;
+		double abs_lower  = (current_interval.lowest > 0) ? 0 : abs(current_interval.lowest);
+		double abs_higher = (current_interval.highest < 0) ? 0 : abs(current_interval.highest);
```

Comment:
Instead of adding a redundant abs here, might be better to change the preceding line to `-current_interval.lowest` ?

Reply:
The `abs` isn't actually redundant. `abs(OverflowSafeInt)` always returns an unsigned integer, so that `abs(T_MIN)` will work correctly.

Reply:
Probably wants a comment to that effect then :)

Reply:
Good idea.